ui/components/app/srp-input/srp-input.js

```diff
@@ -40,7 +44,10 @@ export default function SrpInput({ onChange, srpText }) {
       if (newDraftSrp.some((word) => word !== '')) {
         if (newDraftSrp.some((word) => word === '')) {
           newSrpError = t('seedPhraseReq');
-        } else if (!isValidMnemonic(joinedDraftSrp)) {
+        } else if (
+          isLowerCase(joinedDraftSrp) ||
```

Comment:
Do we want to provide a different error message for cases where the user inputs an uppercase word?  Is "Invalid seed phrase" too generic?

Reply:
Yes, it seems a bit too generic; @SayaGT, Can you please help us with a new appropriate message?

Reply:
@NiranjanaBinoy We could say something like 
1. Invalid input! Secret Recovery Phrase is case sensitive.  OR
2. Input is case sensitive. OR
3. Secret Recovery Phrase is case sensitive.

Reply:
@danjm and @darkwing Shall we proceed with `Invalid input! Secret Recovery Phrase is case sensitive.` as @SayaGT suggested ?